src/NuGet.Indexing/Sql2Lucene.cs

```diff
@@ -223,6 +223,9 @@ public static void Export(string sourceConnectionString, string destinationPath,
             var logger = loggerFactory.CreateLogger<Sql2Lucene>();
             var stopwatch = new Stopwatch();
 
+            // Save the initial time to write to the lucene index
+            var initTime = DateTime.UtcNow;
```

Comment:
Spoke offline about this. We can't use this timestamp. _Most of the time_ this will work out since clock skew from job machine to feed2catalog is likely less than the run time of sql2lucene. However, this is really comparing apples to oranges. Timestamp from job machine has only a weak relationship to timestamp on feed2catalog (which is what catalog2lucene cursors are based off of).

The right value to get here is the `commitTimeStamp` from this document:
https://api.nuget.org/v3/catalog0/index.json